stdlib/http/src/main/ballerina/src/http/client_endpoint.bal

```diff
@@ -345,7 +358,25 @@ public type OutboundAuthConfig record {|
     OutboundAuthHandler authHandler;
 |};
 
-function initialize(string serviceUrl, ClientConfiguration config) returns HttpClient|error {
+# Client configuration for cookies.
+#
+# + enabled - User agents provide users with a mechanism for disabling or enabling cookies
+# + maxPerCookieSize - At least 4096 bytes per cookie (as measured by the sum of the length of the cookie’s name, value, and  attributes)
```

Comment:
The description seems contradictory to what the name of the field suggests. The field name suggests it's the maximum size of a cookie while the description says it's the minimum size

Reply:
Also, wouldn't the field name read better if it's `maxSizePerCookie`?

Reply:
+1 for the changes suggested by @pubudu91 .